src/fidesops/graph/config.py

```diff
@@ -143,15 +144,50 @@ def field_address(self, field: str) -> FieldAddress:
 """An address that corresponds to traversal termination"""
 
 
+class FieldPath:
+    """Fields are addressable by a (possibly) nested name. This key
+    represents a field name held as a tuple of possibly descending keys.
+    A scalar field is represented as a single-element tuple.
+    """
+
+    def __init__(self, *names: str):
+        self.keys = tuple(names)
```

Comment:
FieldPath.keys is a confusing name to me - I think it's too general - as these are descending keys. Calling keys on a dictionary would give you top level keys for example, I just think the term is too overloaded.

Maybe "levels" instead? or similar